.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    "extends": "airbnb-base"
+  };
```

Comment:
Watch your indent here (should match with `m` on line 1), and also include a newline at the end of line 3.

Reply:
Should be resolved.